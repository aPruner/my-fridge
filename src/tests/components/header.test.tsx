import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/header';

test('should render header component correctly', () => {
  const wrapper = shallow(<Header siteTitle="MyFridge" />);
  expect(wrapper).toMatchSnapshot();
});
