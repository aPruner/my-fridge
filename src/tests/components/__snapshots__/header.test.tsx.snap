// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header component correctly 1`] = `
<header
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))
      className="makeStyles-toolBar-2"
    >
      <mockConstructor
        className="makeStyles-title-3"
        to="/"
      >
        <WithStyles(ForwardRef(Typography))
          variant="h6"
        >
          MyFridge
        </WithStyles(ForwardRef(Typography))>
      </mockConstructor>
      <WithStyles(ForwardRef(Button))
        className="makeStyles-signinButton-4"
        color="inherit"
        onClick={[Function]}
      >
        Sign in
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
  <SigninDialog
    onClose={[Function]}
    open={false}
  />
</header>
`;
